docs/examples/ResultsReader.rst

```diff
@@ -1,15 +1,14 @@
+.. |resReader| replace:: :py:class:`~serpentTools.parsers.results.ResultsReader`
 
-Copyright (c) 2017-2018 `Serpent-Tools developer
-team <https://github.com/CORE-GATECH-GROUP/serpent-tools/graphs/contributor>`__,
-GTRC
+.. |homogUniv| replace:: :py:class:`~serpentTools.objects.containers.HomogUniv`
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
-OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
-IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
-CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
-TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
-SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+.. |getUniv| replace:: :py:meth:`~serpentTools.parsers.results.ResultsReader.getUniv`
+
+.. |resdata| replace:: :py:attr:`~serpentTools.parsers.results.ResultsReader.resdata`
+
+.. |metadata| replace:: :py:attr:`~serpentTools.parsers.results.ResultsReader.metadata`
+
+.. |universes| replace:: :py:attr:`~serpentTools.parsers.results.ResultsReader.universes`
 
 Results Reader
 ==============
```

Comment:
Add an empty line and an additional line of `'='` over the 'Results Reader` heading. This causes all other sections to be directly under this chapter. Some of the sections, like `Metadata` are at the same level as this with only a single line of `=`. After building html with `make clean html`, if you navigate to `docs/_build/html/examples/index.html`, then you can see how this affects the layout of this section